### PR TITLE
[SYCL] Add support for sycl::ext::oneapi::property::queue::use_priority

### PR DESCRIPTION
**Pull request by smaslov-intel**
Test for https://github.com/intel/llvm/pull/7523
Signed-off-by: Sergey V Maslov <sergey.v.maslov@intel.com>

### CODE REVIEW

File: SYCL/Plugin/level_zero_queue_priority.cpp [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  // CHECK: [getZeQueue]: create queue {{.*}} priority = Normal
+  test(sycl::property_list{});
+
+  // CHECK: [getZeQueue]: create queue {{.*}} priority = Low
+  test({sycl::ext::oneapi::property::queue::priority_low{}});
+
+  // CHECK: [getZeQueue]: create queue {{.*}} priority = High
+  test({sycl::ext::oneapi::property::queue::priority_high{}});
+
+  // CHECK: Queue cannot be constructed with both of priority_low and priority_high.
```
Comment (smaslov-intel) [inline SYCL/Plugin/level_zero_queue_priority.cpp]:
```suggestion
  // CHECK: Queue cannot be constructed with different priorities.
```

**Comment by smaslov-intel** [comment]
@againull : please review/merge

**Review by againull — APPROVED** [review]

**Pull request merged**